test/unit/test_hub_config.py

```diff
@@ -72,7 +73,25 @@ def tasks_config() -> dict:
                     },
                 ],
                 "submissions_due": {"relative_to": "reference_date", "start": -6, "end": -3},
-            }
+            },
+            {
```

Comment:
Added another round to our test data so we can test the new function for multiple rounds.